konig-data-app-gcp/src/main/java/io/konig/data/app/gae/GaDataTransformServlet.java

```diff
@@ -0,0 +1,54 @@
+package io.konig.data.app.gae;
+
+/*
+ * #%L
+ * Konig Data App for Google Cloud Platform
+ * %%
+ * Copyright (C) 2015 - 2017 Gregory McFall
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+
+import javax.servlet.ServletException;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+
+import io.konig.data.app.common.DataTransformServlet;
+import io.konig.sql.runtime.BigQueryDataTransformService;
+
+public class GaDataTransformServlet extends DataTransformServlet {
+	private static final long serialVersionUID = 1L;
+	private static final String CREDENTIALS_FILE = "konig/gcp/credentials.json";
```

Comment:
We now have several components that depend on this file location.  We should move this static variable into a shared utility class so that it is defined only once.